docs/intro/tutorial.rst

```diff
@@ -55,318 +55,333 @@ This will create a ``tutorial`` directory with the following contents::
+            <a class="tag" href="/tag/world/page/1/">world</a>
         </div>
     </div>
 
+Let's open up scrapy shell and play a bit to find out how to extract the data
+we want::
 
-So we can select each ``<div class="quote">`` element belonging to the site's 
-list with this code::
+    $ scrapy shell http://quotes.toscrape.com
 
-    response.xpath('//div[@class="quote"]')
+We get a list of selectors for the quote HTML elements with::
 
-From the quote elements, we can select the texts with::
+    >>> response.css("div.quote")
 
-    response.xpath('//div[@class="quote"]/span[@class="text"]/text()').extract()
+Each of the selectors returned by the query above allows us to run further
+queries over their sub-elements. Let's assign the first selector to a
+variable, so that we can run our CSS selectors directly on a particular quote::
 
-The authors::
+    >>> quote = response.css("div.quote")[0]
 
-    response.xpath('//div[@class="quote"]/span/small/text()').extract()
+Now, let's extract ``title``, ``author`` and the ``tags`` from that quote
+using the ``quote`` object we just created::
 
-As we've said before, each ``.xpath()`` call returns a list of selectors, so we can
-concatenate further ``.xpath()`` calls to dig deeper into a node. We are going to use
-that property here, so::
+    >>> title = quote.css("span.text::text").extract_first()
+    >>> title
+    '“The world as we have created it is a process of our thinking. It cannot be changed without changing our thinking.”'
+    >>> author = quote.css("small.author::text").extract_first()
+    >>> author
+    'Albert Einstein'
 
-    for quote in response.xpath('//div[@class="quote"]'):
-        text = quote.xpath('span[@class="text"]/text()').extract()
-        author = quote.xpath('span/small/text()').extract()
-        print('{}: {}'.format(author, text))
+Given that the tags are a list of strings, we can use the ``.extract()`` method
+to get all of them::
 
-.. note::
+    >>> tags = quote.css("div.tags a.tag::text").extract()
+    >>> tags
+    ['change', 'deep-thoughts', 'thinking', 'world']
+
+Having figured out how to extract each bit, we can now iterate over all the
+quotes elements and put them together into a Python dictionary::
+
+    >>> for quote in response.css("div.quote"):
+    ...     text = quote.css("span.text::text").extract_first()
+    ...     author = quote.css("small.author::text").extract_first()
+    ...     tags = quote.css("div.tags a.tag::text").extract()
+    ...     print(dict(text=text, author=author, tags=tags))
+    {'text': u'\u201cThe world as we have created it is a process of our thinking. It cannot be changed without changing our thinking.\u201d', 'tags': [u'change', u'deep-thoughts', u'thinking', u'world'], 'author': u'Albert Einstein'}
+    {'text': u'\u201cIt is our choices, Harry, that show what we truly are, far more than our abilities.\u201d', 'tags': [u'abilities', u'choices'], 'author': u'J.K. Rowling'}
```

Comment:
In Python 3 users won't get those ugly `\u201cIt`. What do you think about using Python 3 in the tutorial, and mentioning that in Python 2 output can be different? I'm not sure about that.